src/deployment/deployment.service.ts

```diff
@@ -33,6 +33,7 @@ export interface Deployment {
   multicallAddress: string;
   gasToken: GasToken;
   startBlock: number;
+  networkId: number;
```

Comment:
does this belong here? I see places in the code where different networkId values are passed for the same deployment